addon-perf/index.html

```diff
@@ -12,13 +12,26 @@
     <script type="text/javascript" src="/lib/jquery.csv-0.8.2.js"></script>
     <link rel="stylesheet" href="addon-perf.css" />
     <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.4/themes/smoothness/jquery-ui.css">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-49796218-2"></script>
     <script>
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-      ga('create', 'UA-49796218-2', 'mozilla.org');
-      ga('send', 'pageview');
+      var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+      var fx = navigator.userAgent.match(/Firefox\/(\d+/);
+      var ie10 = navigator.userAgent.match(/MSIE 10/i);
+      var w8 = navigator.appVersion.match(/Windows NT 6.2/);
+      if (fx && Number(fx[1]) < 32) {
+        _dntStatus = 'Unspecified'; // bug 887703
+      } else if (ie10 && w8) {
+        _dntStatus = 'Unspecified';
+      } else {
+        _dntStatus = {'0': 'Disabled', '1': 'Enabled'}[_dntStatus] || 'Unspecified';
+      }
+      if (_dntStatus !== 'Enabled') {
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', 'UA-49796218-2');
+      }
```

Comment:
As I said in my review of the previous pr, I think we should factor this function into a seperate file and import it where needed for clarity.